docs/supporting-artifacts/0030-LMS-ForeignKeys.sql

```diff
@@ -44,6 +44,15 @@ CREATE NONCLUSTERED INDEX [FK_LMSGrade_LMSUserLMSSectionAssociation]
 ON [lms].[LMSGrade] ([LMSSectionIdentifier] ASC, [LMSUserIdentifier] ASC, [LMSUserLMSSectionAssociationIdentifier] ASC)
 GO
 
+ALTER TABLE [lms].[LMSSection] WITH CHECK ADD CONSTRAINT [FK_LMSSection_CourseOffering] FOREIGN KEY ([EdFiLocalCourseCode], [EdFiSchoolId], [EdFiSchoolYear], [EdFiSessionName])
+REFERENCES [edfi].[CourseOffering] ([LocalCourseCode], [SchoolId], [SchoolYear], [SessionName])
```

Comment:
We don't want to have a foreign key - remember, the LMS Toolkit needs to be useable without an ODS.

Reply:
This was just the generated supporting artifact, I didn't bring it over to the migration. Academic now.